recipes/aws-crt-cpp/all/conanfile.py

```diff
@@ -42,18 +41,23 @@ def validate(self):
 
     def requirements(self):
         self.requires("aws-c-event-stream/0.2.7")
-        self.requires("aws-c-common/0.6.9")
-        self.requires("aws-c-io/0.10.9")
-        self.requires("aws-c-http/0.6.7")
-        self.requires("aws-c-auth/0.6.4")
-        self.requires("aws-c-mqtt/0.7.6")
-        self.requires("aws-c-s3/0.1.19")
-        self.requires("aws-checksums/0.1.11")
+        self.requires("aws-c-common/0.6.15")
+        self.requires("aws-c-io/0.10.13")
+        self.requires("aws-c-http/0.6.10")
+        self.requires("aws-c-auth/0.6.8")
+        self.requires("aws-c-mqtt/0.7.9")
+        self.requires("aws-c-s3/0.1.29")
+        self.requires("aws-checksums/0.1.12")
 
     def source(self):
         tools.get(**self.conan_data["sources"][self.version],
             destination=self._source_subfolder, strip_root=True)
 
+    def export_sources(self):
+        self.copy("CMakeLists.txt")
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            self.copy(patch["patch_file"])
```

Comment:
try to keep execution order, it's quite misleading to see export_sources() after source(), actually it's completly unrelated.

Reply:
Sorry, I forget this documents.
https://github.com/conan-io/conan-center-index/blob/master/docs/reviewing.md#order-of-methods-and-attributes

I will fix it.